apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/BuiltInFunctionResourceIntegrationTest.java

```diff
@@ -411,11 +411,9 @@ public void setUpdateManyValuesBaseDoc() {
           .get(BASE_PATH, DEFAULT_NAMESPACE, DEFAULT_COLLECTION, DOCUMENT_ID)
           .then()
           .body("documentId", equalTo(DOCUMENT_ID))
-          .body("data", jsonPartEquals("b[1].nested", "value"))
```

Comment:
Why were these removed?

Reply:
This entire test fn was added - but then I realized that because it is updating data on the root of the document, the data at the root is just different.

In the test this was copied from, I was setting data on a nested path, and then checking that both the data at the nested path previously and the new data were both present.

Reply:
Ok